src/workers/hermes_worker.js

```diff
@@ -37,6 +39,18 @@ export default class HermesWorker extends PeriodicWorker {
     this.expressApp.get('/health', asyncMiddleware(
       healthCheckHandler(mongoClient, dataModelEngine.identityManager.web3)
     ));
+    const registry = new promClient.Registry();
+    this.expressApp.get('/metrics', prometheusMetricsHandler(registry));
+    this.totalBundlesUploaded = new promClient.Counter({
```

Comment:
Again this is something I would love to see inside of a separate `Tracking` service. With a very small interface that hides all the prometheus (and possible other technologies) away behind very simple methods like `markBundleUploadSuccess` and `markBundleUploadFailed`.

Reply:
What is the benefit of that? We would just be moving complexity to a different file.